sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -447,16 +446,6 @@ fn verify_outputs<const VERIFY: bool>(outputs: &[Output], visitor: &ProtocolPara
                 return Err(Error::InvalidTransactionAmountSum(amount_sum as u128));
             }
 
-            if let Some(native_tokens) = native_tokens {
-                native_tokens_count = native_tokens_count.checked_add(native_tokens.len() as u8).ok_or(
-                    Error::InvalidTransactionNativeTokensCount(native_tokens_count as u16 + native_tokens.len() as u16),
-                )?;
-
-                if native_tokens_count > NativeTokens::COUNT_MAX {
```

Comment:
This check is still listed in https://github.com/iotaledger/tips/blob/tip45/tips/TIP-0045/tip-0045.md
Really not needed anymore?

Reply:
I've commented, we definitely said we'd remove it